library/std/Cargo.toml

```diff
@@ -104,7 +104,6 @@ level = "warn"
 # for rust-lang/rust. But for users of `-Zbuild-std` it does.
 check-cfg = [
     'cfg(bootstrap)',
-    'cfg(target_arch, values("xtensa"))',
```

Comment:
This was added about a month ago, so probably will be needed at least until the next beta bump (?).

Reply:
yeah, my bad, I think I was the one to add it, but for some reason it didn't failed locally

also `xtensa` is a bit weird since we don't (currently?) have a any target with that arch